src/appleseed/renderer/modeling/input/inputarray.h

```diff
@@ -53,7 +53,7 @@ namespace renderer
 // Input formats.
 //
 
-enum InputFormat
```

Comment:
That's a good change but it obscures the main changes of this PR. If possible, I would suggest to split this into its own commit.

Reply:
it won't compile